cocos/particle/noise.ts

```diff
@@ -106,34 +120,82 @@ export class ParticleNoise {
     private result: Vec3 = new Vec3();
     private mixOut: Vec2 = new Vec2();
 
+    /**
+     * @en Set texture rolling speed
+     * @zh 设置纹理滚动速度
+     * @param x @en X axis roll speed @zh X 轴滚动速度
+     * @param y @en Y axis roll speed @zh Y 轴滚动速度
+     * @param z @en Z axis roll speed @zh Z 轴滚动速度
+     */
     public setSpeed (x, y, z) {
         this.noiseSpeed.set(x, y, z);
     }
 
+    /**
+     * @en Set noise frequency
+     * @zh 设置生成的噪声频率
+     * @param f @en Noise texture frequency @zh 噪声频率
+     */
     public setFrequency (f) {
         this.noiseFrequency = f;
     }
 
+    /**
+     * 
+     * @param x 
+     * @param y 
+     * @param z 
+     * @deprecated since v3.6.0
+     */
     public setAbs (x, y, z) {
         this.noiseAbs.set(x, y, z);
     }
 
+    /**
+     * @en Set noise amplititude
+     * @zh 设置噪声强度
+     * @param x @en Noise amplititude on X axis @zh X 轴上的噪声强度
+     * @param y @en Noise amplititude on Y axis @zh Y 轴上的噪声强度
+     * @param z @en Noise amplititude on Z axis @zh Z 轴上的噪声强度
+     */
     public setAmplititude (x, y, z) {
         this.noiseAmplitude.set(x, y, z);
     }
 
+    /**
+     * @en Set noise texture layer count
+     * @zh 设置噪声纹理层数
+     * @param x @en Layer count on X axis @zh X 轴方向的噪声层数
+     * @param y @en Layer count on Y axis @zh Y 轴方向的噪声层数
```

Comment:
Seems not correct according your getNoise, the Y seems to be the `Octave multiplier` and Z seems to be the `Octave scale`